virttest/utils_libvirtd.py

```diff
@@ -119,14 +119,15 @@ def stop(self):
             result.append(daem_item.stop())
         return all(result)
 
-    def restart(self, reset_failed=True):
+    def restart(self, reset_failed=True, wait_for_start=True):
```

Comment:
Could you also include start() for this enhancement?

Reply:
Thanks dzheng, Done